iree/compiler/Translation/test/smoketest.mlir

```diff
@@ -1,4 +1,4 @@
-// RUN: iree-translate -split-input-file --iree-input-type=mhlo -iree-hal-target-backends=vmvx -iree-mlir-to-vm-bytecode-module -iree-vm-bytecode-module-output-format=flatbuffer-text %s | FileCheck %s
+// RUN: iree-compile -split-input-file --iree-input-type=mhlo -iree-hal-target-backends=vmvx -iree-mlir-to-vm-bytecode-module -iree-vm-bytecode-module-output-format=flatbuffer-text %s | FileCheck %s
```

Comment:
`compiler/Translation/test/` should probably keep using `iree-translate`? Or should it test both?
Same question for `iree/compiler/Translation/test/streams.mlir`

Reply:
For the moment, it should switch. This directory is mixing a couple of things that need to be pried apart in a followup.